synapse/tests/test_lib_remcycle.py

```diff
@@ -523,6 +523,7 @@ def ondone(job_tufo):
 
     def test_hypnos_throw_timeouts(self):
         # Run a test scenario which will generate hundreds of jobs which will timeout.
+        self.skipTest(reason='This test can periodically cause coverage.py failures and even then may not run sucessfully.')
```

Comment:
I agree. Let's skip this for now. We had a separate discussion about this kind of integration tests